src/test/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,49 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App snapshot 1`] = `
+<div>
+  <div
+    className="sc-htoDjs MmwNX"
```

Comment:
this won't work. Use jest styled components as the snapshots will fail everytime